src/lib/field/field.ts

```diff
@@ -1,14 +1,16 @@
-import { FieldDensityType, FieldFloatLabelType, FieldShapeType, FIELD_CONSTANTS } from './field-constants';
+import { coerceBoolean, ensureChildren, FoundationProperty } from '@tylertech/forge-core';
+
 import { BaseComponent, IBaseComponent } from '../core/base/base-component';
+import { FIELD_CONSTANTS, FieldDensityType, FieldFloatLabelType, FieldShapeType } from './field-constants';
 import { FieldFoundation } from './field-foundation';
-import { coerceBoolean, ensureChildren, FoundationProperty } from '@tylertech/forge-core';
 
 export interface IFieldComponent extends IBaseComponent {
   density: FieldDensityType;
   floatLabelType: FieldFloatLabelType;
   shape: FieldShapeType;
   invalid: boolean;
   required: boolean;
+  addonEndAlwaysEnabled: boolean;
```

Comment:
I'm not in love with this name, but I also can't think of anything better. Naming is hard... haha

I assume you went through the same thought process and this seems reasonable, but I'll think on it for a min and see if anything comes to mind otherwise we can go with this.

Reply:
Yea, I ran through a couple different names and this seemed to be the least terrible. As you said, naming stuff is hard.